guide/index.adoc

```diff
@@ -74,7 +74,7 @@ The Seed CLI is heavily reliant on Docker to build and package Seed images (Dock
 
 [WARNING]
 ====
-If using Windows, we strongly recommend using Seed with {wsl2-url}[WSL 2] rather than native Windows to avoid incompatibilities between environments. 
+If using Windows, we strongly recommend using Seed with {wsl2-url}[WSL 2]. 
```

Comment:
> If using Windows, we strongly recommend using Seed with {wsl2-url}[WSL 2] in order to avoid incompatibilities between environments.

I think giving the reasoning would still be good to include.

Reply:
Makes sense. There is a similar warning on line 821 that I also updated.